src/common/eeprom.h

```diff
@@ -212,6 +214,25 @@ extern void eeprom_clear(void);
 // PUT test
 int8_t eeprom_test_PUT(const unsigned int);
 
+/**
+ * @brief reads Z offset, directly or from current sheet if sheets are enabled
+ *
+ * @return float Z offset
+ */
+float eeprom_get_z_offset();
+
+/**
+ * @brief writes Z offset, directly or to current sheet if sheets are enabled
+ *
+ * @param value value to set
+ * @return true successfully set
+ * @return false sheet index stored in eeprom corrupted
+ */
+bool eeprom_set_z_offset(float value);
```

Comment:
This function has return value but it is not checked anywhere it is used.

Reply:
added return value checks